src/rgw/Makefile.am

```diff
@@ -120,6 +120,11 @@ LIBRGW_DEPS += \
 	-ldl
 endif
 
+if FREEBSD
```

Comment:
On 22-3-2016 04:19, Kefu Chai wrote:
> In src/rgw/Makefile.am
> <https://github.com/ceph/ceph/pull/7515#discussion_r56932517>:
> 
>> @@ -120,6 +120,11 @@ LIBRGW_DEPS += \
>>  	-ldl
>>  endif
>>  
>> +if FREEBSD
> 
> guard this with "if WITH_OPENLDAP" also.

Correct,

I have not yet rebased to the version where all the LDAP dust has
settled. So that definitely needs to be there.

--WjW